### DISCUSSION TITLE
Importing Playlists from a CSV?

### DISCUSSION THREAD

**Discussion by BeeTeeBeats** [FreeTubeApp/FreeTube#6308] [Q&A]
I am new to FreeTube, and I was curious about importing playlists. I can import subscriptions just fine, but playlists are expected to be a .db file, and YouTube only downloads playlists as a .csv. Is there a way to import the csv or a way to convert the csv to the right type of db file?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @BeeTeeBeats, importing from YouTube isnt possible at the moment. There is an open issue for this https://github.com/FreeTubeApp/FreeTube/issues/5057. The .`db` file is FreeTube own file for exporting/importing your local playlists to another machine
